src/Expressions/BinaryOperatorExpression.php

```diff
@@ -226,6 +226,9 @@ public function evaluate(row $row, AsyncMysqlConnection $conn): mixed {
           $pattern = Str\strip_suffix($pattern, '%');
         }
 
+        // escape all + characters
+        $pattern = \preg_quote($pattern, '+');
```

Comment:
The 2nd argument is for the pattern delimiter at the start and end of the regex which is `/` below (on line 236). I think it should be `/` here too.

It will always escape all the regex special characters including `+`.

Maybe adding a test case that uses a like query with a `/` in it would be helpful to test this too?

Reply:
@ssandler I think I've updated to do what you suggested, but please let me know if I've misunderstood.